osf/models/metaschema.py

```diff
@@ -158,19 +159,16 @@ def absolute_api_v2_url(self):
 
 class RegistrationFormBlock(ObjectIDMixin, BaseModel):
     class Meta:
-        unique_together = ('schema', 'block_id')
         order_with_respect_to = 'schema'
 
     schema = models.ForeignKey('RegistrationSchema', related_name='form_blocks', on_delete=models.CASCADE)
-    page = models.CharField(max_length=255)
-    section = models.CharField(max_length=255, null=True)
     help_text = models.TextField()
```

Comment:
Many descriptions from the original schema end up getting mapped to `help_text`, as they were meant to aid the user filling out the form, and not formatted for display to all users.